pkg/question/input.go

```diff
@@ -2,11 +2,28 @@ package question
 
 import (
 	"fmt"
-
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/spf13/cobra"
 )
 
+const FlagConfirm = "confirm"
+
+type DeleteFlags struct {
+	Confirm *flag.Flag[bool]
+}
+
+func NewDeleteFlags() *DeleteFlags {
+	return &DeleteFlags{
+		Confirm: flag.New[bool](FlagConfirm, false),
+	}
+}
+
+func RegisterDeleteFlag(value *bool, cmd *cobra.Command, resourceDescription string) {
+	cmd.Flags().BoolVarP(value, FlagConfirm, "c", false, fmt.Sprintf("Don't ask for confirmation before deleting the %s.", resourceDescription))
```

Comment:
Should confirm have a short flag of `-c` here?
Short flags are designed to make it quick and easy to do something (e.g. -p for project because you specify it all the time). "short and easy" doesn't feel quite right for deleting stuff tho?
@jbristowe is the arbiter of this kind of thing, suggest getting his opinion if you haven't already.

Reply:
actually the shorthand was `y`, not `c`